src/main/java/uk/gov/hmcts/darts/arm/client/model/rpo/RemoveProductionRequest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.darts.arm.client.model.rpo;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@Builder
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Suggest you remove `@JsonIgnoreProperties` has it doesn't have much value for a request object that we're manually constructing.